job-descriptions/software-engineer-frontend.md

```diff
@@ -0,0 +1,66 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# Software Engineer - Frontend
+
+### About us
+
+We're building the new standard developer platform. Top tech companies have invested \$100Ms to build internal developer platforms for code search, code review, alerts, and automation. Sourcegraph provides this standard developer platform to every company, helping startups and large enterprises ship better software faster.
+
+[Our mission](https://sourcegraph.com/plan) is to dramatically increase the number of people who can understand and write code. By making code more accessible, we will democratize software development and accelerate innovations that bring the future sooner in transportation, health care, energy, AI, communication, space travel, etc.
+
+We value openness and transparency. [Our code](https://github.com/sourcegraph/sourcegraph), [our roadmap](https://about.sourcegraph.com/direction), and [our company processes](https://about.sourcegraph.com/company/open_source_open_company) are public. This transparency helps us rapidly gather feedback from our customers so we can iterate, learn, and deliver the best product.
+
+To create a product that serves the needs of all developers, we are building a diverse remote-first team that is distributed across the world. Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
+
+If you are passionate about making the world better through software, come join us!
+
+### About the role
```

Comment:
Right now, this section lists skills, but doesn't give a lot of concrete examples of projects the candidate would be working on if hired. Do we assume that they should read through our roadmap to find this out? If so, should we link to the roadmap more prominently here, rather than just bundling it in the "openness and transparency" section above?

Reply:
I was debating whether it would be redundant or not, but since you mentioned it I added another link specifically to the web section of the project roadmap in fd8e2f430835caff448ea26a1ed14ac681afbc3d.